bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/config/Clip2ThingConfig.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hue.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * Configuration for CLIP V2 things.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ */
+@NonNullByDefault
+public class Clip2ThingConfig {
+    public @Nullable String resourceId;
```

Comment:
This would simplify handling and decrease risk of `NullPointerException` since parameter is required:
```suggestion
    public String resourceId = "";
```

Reply:
Ok. Done.